wntr/epanet/io.py

```diff
@@ -777,7 +777,8 @@ def _write_pumps(self, f, wn):
             else:
                 raise RuntimeError('Only head or power info is supported of pumps.')
             tmp_entry = _PUMP_ENTRY
-            if pump.speed_timeseries.base_value != 1:
+            #if pump.speed_timeseries.base_value != 1:
+	    if pump.speed_timeseries.base_value is not None:
```

Comment:
This looks good, but the spacing is wrong (it has a tab instead of all spaces) which is causing the tests to fail. Can you fix that to be spaces with the right indenting and resubmit?

Reply:
ok. #SiliconValley :)